### ISSUE TITLE
Header Sign-in functionality

### ISSUE THREAD

**Issue by amol-anand** [hlxsites/caesars-careers#5]
Update header to match caesars.com/careers/ site

**Comment by amol-anand** [comment]
Working on fixing the CSS alignment and removing unnecessary pieces from the header that are no longer needed due to the way Sign In will work

**Comment by amol-anand** [comment]
Still need to implement sign in functionality. Waiting on Caesars team for the info
